cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -43,3 +70,37 @@ def test_non_admin_user_reports_access_rest(appliance, request, rbac_api):
     report_data = rbac_api.collections.reports.all
     assert_response(appliance)
     assert len(report_data)
+
+
+@pytest.mark.tier(1)
+def test_reports_custom_tags(appliance, request, create_custom_tag):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: low
+        initialEstimate: 1/3h
+        setup:
+            1. Add custom tags to appliance using black console
+                i. ssh to appliance, vmdb; rails c
+                ii. cat = Classification.create_category!(
+                    name: "rocat1", description: "read_only cat 1", read_only: true)
+                iii. cat.add_entry(name: "roent1", description: "read_only entry 1")
+        testSteps:
+            1. Create a new report with the newly created custom tag/category.
+        expectedResults:
+            1. Report must be created successfully.
+    """
+    category_name = create_custom_tag
+    report_data = {
```

Comment:
Optional: Thoughts on moving this into a fixture?

Reply:
Creating the report is a part of the test and creating a separate fixture that just returns the dictionary data seems a little futile imo. But if in the future, I add more test cases that require report data, I will move it to fixture.